tests/eclient/testdata/profile.txt

```diff
@@ -0,0 +1,106 @@
+# Test local manager and profiles
+
+{{define "ssh"}}ssh -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa root@{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with a 1 reboot limit
+! test eden.reboot.test -test.v -timewait 40m -reboot=0 -count=1 &
+
+# Define local-manager and two apps in different profiles
+eden pod deploy -n local-manager --memory=512MB docker://itmoeve/eclient:0.7 -p 2223:22
+eden pod deploy -n app-profile-1 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
+eden pod deploy -n app-profile-2 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
+
```

Comment:
Somewhere we need to have apps which can run in two configs.I think that means an app instance can be in more than one profile. Thus a third app instance  which is in profile-1 and profile-2 (and then it you switch to "profile-3" they both halt.

Reply:
Ok, will add it.